redislock_test.go

```diff
@@ -311,3 +393,14 @@ func teardown(t *testing.T, rc *redis.Client) {
 		t.Fatal(err)
 	}
 }
+
+func getCallingFunctionName(skipFrameCount int) string {
+
```

Comment:
Nit: weird line skip.

Doesn't prevent us from merging and then polishing it ourselves.

Reply:
removed